source/MissionAction.h

```diff
@@ -72,26 +74,30 @@ class MissionAction {
 	std::string system;
 	LocationFilter systemFilter;
 	
-	std::string logText;
-	std::map<std::string, std::map<std::string, std::string>> specialLogText;
-	
 	std::string dialogText;
 	const Phrase *stockDialogPhrase = nullptr;
 	Phrase dialogPhrase;
 	
 	const Conversation *stockConversation = nullptr;
 	Conversation conversation;
 	
-	std::map<const GameEvent *, std::pair<int, int>> events;
 	std::map<const Outfit *, int> gifts;
 	std::map<const Outfit *, int> requiredOutfits;
+	
+	GameAction actions;
+	/* Now handled by GameAction
```

Comment:
This out-commented code makes sense during the PR, but will just be out-commented code after this gets merged. I suggest removing all the out-commented code (and maybe put some comment above GameAction on what the GameAction handles).